recommenders/popularity_recommender.py

```diff
@@ -1,26 +1,28 @@
-from recommenders import data_movies, data_ratings
-import pandas as pd
-
-
-def recommend_popular_movies():
```

Comment:
There is no function docstring provided. It is recommended to include a docstring at the beginning of every function to describe its purpose and how it should be used.